analyzer/pgn.h

```diff
@@ -6540,7 +6540,7 @@ Pgn pgnList[] = {
      PACKET_FAST,
      {COMPANY(419),
       MATCH_LOOKUP_FIELD("Message ID", BYTES(2), 32813, FUSION_STATUS_MESSAGE_ID),
-      UINT8_PRIMARY_KEY_FIELD("Source"),
+      UINT8_PRIMARY_KEY_FIELD("Number"),
```

Comment:
I don't like field with the name "number" here. Too much meaningless in my mind.